src/Models/SupervisorShift.php

```diff
@@ -55,4 +56,12 @@ public function jobSiteVisits(): HasMany
     {
         return $this->hasMany(JobSiteVisit::class);
     }
+
+    public function getIsActiveAttribute(): bool
+    {
+        if (!$this->end_time && $this->start_time &&  $this->start_time >= Carbon::now()->subDay()) {
```

Comment:
Avoid if true returne true, else return false, just return the expression

Reply:
done